test/NCI.OCPL.Api.CTSListingPages.Tests/Tests/TestDataObjects/ESLabelLookupQueryService/Get_BaseScenario.cs

```diff
@@ -0,0 +1,22 @@
+using Newtonsoft.Json.Linq;
+
+namespace NCI.OCPL.Api.CTSListingPages.Tests
+{
+    public abstract class Get_BaseScenario
+    {
+        /// <summary>
+        /// The mock response from Elasticsearch.
+        /// </summary>
+        /// <value></value>
+        public abstract string MockESResponse { get; }
+
+        /// <summary>
+        /// The LabelInformation which the service is expected to return from the response.
+        /// </summary>
+        public abstract LabelInformation ExpectedData { get; }
+
+        public abstract string ErrorMessage { get; }
+
+        public abstract Moq.Times TimesCalled { get; }
```

Comment:
Please replace this with an integer and rename to reflect that it's the maximum number of times that _Logging_ is expected to be called.

Also, please add comment blocks for both this member, and `ErrorMessage`.